Atualização de notificações e tradução

fernandoacosta opened the pull request:
Fiz um update na notificação que vem do CobreGrátis que não estava pegando o ID do pedido e, portanto, não atualizava o status para Processando.

Além disso, a tradução oficial não está funcionando. Então fiz um update simples nela e já funcionou. Aproveitei e modifiquei alguns termos.

fernandoacosta committed:
Update order_id on notification

Todos os pedidos retornam "order-cobregratis" como $data['meta']. Então modifiquei a order-id para $data[document_number], que é o número do pedido.

fernandoacosta committed:
Update translation

A tradução não estava funcionando. Apenas atualizei com POEdit.
Aproveitei e modifiquei alguns termos.

claudiosanches commented:
Não podemos usar document_number, vai dar problemas com plugins que alteram o filtro `woocommerce_order_number`. Em outras palavras, não é o ID do pedido isso e sim o número.

O correto é o campo meta:

``` php
'meta' => 'order-' . $order->id
```

Que realmente pega o ID e por isso validamos com:

``` php
$order_id = intval( str_replace( 'order-', '', $data['meta'] ) );
```

fernandoacosta committed:
Update order_id

O valor de 'meta' sempre retornava o padrão order-cobregratis. Agora
pega o ID corretamente.

fernandoacosta commented:
Entendi!

'meta'                 => 'order-' . $this->id sempre retorna  "order-cobregratis". Só funciona corretamente se alterado para $order->id
